Makefile

```diff
@@ -126,9 +127,9 @@ all: manager ## Build all binaries
 build-licenses.csv: go.mod ## Track licenses in a CSV file
 	@echo "Tracking licenses into file $@"
 	@echo "========================================"
-	GOOS=linux GOARCH=amd64 go mod download
-	# https://github.com/google/go-licenses/issues/244
-	GOTOOLCHAIN=local GOOS=linux GOARCH=amd64 $(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > licenses.csv
+	export GOOS=linux
+	export GOARCH=amd64
+	go run github.com/google/$(GO_LICENSES)@v$(GO_LICENSES_VERSION) csv --include_tests $(BASE_GO_PACKAGE)/... > licenses.csv
```

Comment:
Not the prettiest fix, but the only one I could find. 

From here:
https://github.com/google/go-licenses/issues/149#issuecomment-2208882223

Seems we won't be able to use devbox/nix for the go-licenses binary and instead need to use it as a module dep. 

cc @roothorp